test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -2931,48 +2951,73 @@ TEST_F(WeightedRoundRobinTest, Basic) {
             channel->GetLoadBalancingPolicyName());
 }
 
-TEST_F(WeightedRoundRobinTest, OobReporting) {
+INSTANTIATE_TEST_SUITE_P(WeightedRoundRobin, WeightedRoundRobinParamTest,
```

Comment:
Please move this up to line 2916, so that it's easy to see how the test suite is being parameterized right after it is defined.

Reply:
Done.